python-ufonormalizer/PKGBUILD

```diff
@@ -4,37 +4,38 @@
 pkgname=python-ufonormalizer
 _pyname=${pkgname#python-}
 pkgver=0.6.1
-pkgrel=3
-pkgdesc='A tool that will normalize XML and other data inside of a UFO'
+pkgrel=6
+pkgdesc="A tool that will normalize XML and other data inside of a UFO"
```

Comment:
Please don't change the coding style just for the sake of changing it. This is a fixed string with no substitutions, so single quotes are good.